python-sdk/docs/development/RELEASE.md

```diff
@@ -2,7 +2,9 @@
 
 ## Updating the changelog
 
-The first step to creating a release is identifying what changed from the previous release and adding a description to the [CHANGELOG.md](../CHANGELOG.md)
+The first step to creating a release is identifying what changed from the previous release and adding a description to the [CHANGELOG.md](../CHANGELOG.md).
+
+One way to identify what changed from the previous release is to raise a dummy PR from the main branch to the previous release branch. This will show all the commits that are in the main branch but not in the release branch. Using this we can note all the changes significant enough to be mentioned in CHANGELOGS.
```

Comment:
I think using the comparing URL is better..WDYT ..for example to compare `main` to `1.3.3`

`https://github.com/astronomer/astro-sdk/compare/main...1.3.3`

Reply:
I like @phanikumv 's suggestion, it is always good to avoid dummy PRs if we can